activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+*   Add `update_sql` option to `#upsert_all` to make it possible to use raw SQL to update columns on conflict:
+
+    ```ruby
+    Book.upsert_all(
+      [{ id: 1, status: 1 }, { id: 2, status: 1 }],
+      update_sql: "status = GREATEST(books.status, EXCLUDED.status)"
```

Comment:
Instead of `update_sql` does it makes sense to use the `on_duplicated` option? If it is a string we use it as the SQL to update on conflict.